### PR TITLE
:zap: Mt 882 add h4 tag title card article

### PR DESCRIPTION
**Pull request by miloszhygen**

### CODE REVIEW

File: src/svelte/CardArticle.svelte [hunk 1]
```diff
@@ -6,12 +6,14 @@
...
   export let type: 'task' | 'form-intro-module' | 'legal-text' | '' = ''
-  export let headerTag: 'h2' | 'h3' = 'h3'
+  export let headerTag: 'h2' | 'h4' | 'h3' = 'h3' // Fallback is h3
 </script>
 
 <article class={type} aria-labelledby={headingId}>
   {#if headerTag === 'h2'}
     <h2 id={headingId} class="title h3">{@html title}</h2>
+  {:else if headerTag === 'h4'}
+    <h4 id={headingId} class="title">{@html title}</h4>
```
Comment (lha047) [inline src/svelte/CardArticle.svelte:16]: Did you have this throught desing review?
Looks like the styling is a bit different. Maybe use the same class `h3` as for the h2 tag?

**Review by lha047 — CHANGES_REQUESTED** [review]
See my comment :)

File: src/svelte/CardArticle.svelte (hunk 1 above)
Comment (miloszhygen) [reply src/svelte/CardArticle.svelte:16]: I'll test, and take it through design review :)

**Comment by miloszhygen** [comment]
Added the class h4 to the h4 tag after the design review

**Review by lha047 — APPROVED** [review]
Good job :D

**Pull request merged**